### DISCUSSION TITLE
Migrate Modernjs modules to rslib

### DISCUSSION THREAD

**Discussion by alexandesigner** [web-infra-dev/rslib#190] [General]
Hello everyone, I have some modules using [modernjs module ](https://modernjs.dev/module-tools/en/) to build the packages, I have seen similar features in both libs (modernjs modele and rslib), should I wait for a more stable version to do this "migration" and at some point will modernjs incorporate rslib?

**Comment by fi3ework** [comment]
> should I wait for a more stable version to do this "migration" and at some point will modernjs incorporate rslib

Currently, Rslib will become the next generation of Modern.js Module, with the vast majority of capabilities being supported (hooks won't, Modern.js Module's hooks are based on esbuild, Rslib will provide more powerful hooks mechanism based on Rspack and Rsbuild). We will provide the guide to migrate Modern.js Module to Rslib.
